local-volume/bootstrapper/deployment/kubernetes/config-gce-local-ssd.yaml

```diff
@@ -1,5 +1,6 @@
 # The config map is used to configure local volume discovery for Local SSDs on GCE and GKE. 
 # It is a map from storage class to its mount configuration.
+apiVersion: v1
```

Comment:
You need to update the configs below from json to yaml

Reply:
done